t_s3/test_missed_headers.py

```diff
@@ -0,0 +1,190 @@
+import string
+
+from test_suite import marks, tester
+
+__author__ = "Tempesta Technologies, Inc."
+__copyright__ = "Copyright (C) 2018-2025 Tempesta Technologies, Inc."
+__license__ = "GPL2"
+
+
+class CustomTemplate(string.Template):
+    delimiter = "&"
+
+
+@marks.parameterize_class(
+    [
+        {"name": "MethodHEADStatus200", "method": "HEAD", "status": "200"},
+        {"name": "MethodPOSTStatus200", "method": "POST", "status": "200"},
+        {"name": "MethodDeleteStatus200", "method": "DELETE", "status": "200"},
+        {"name": "MethodDeleteStatus204", "method": "DELETE", "status": "200"},
+        {"name": "MethodPatchStatus200", "method": "PATCH", "status": "200"},
+        {
+            "name": "MethodPUTStatus200",
+            "method": "PUT",
+            "status": "200",
+        },
+        {
+            "name": "MethodPUTStatus204",
+            "method": "PUT",
+            "status": "200",
+        },
+    ]
+)
+class TestMissedContentLengthInMethod(tester.TempestaTest):
```

Comment:
Probably should we add tests for 3xx, 4xx, 5xx responses? Most likely, CT and CL headers are processed in a different way.